modules/tinymce/src/plugins/textpattern/main/ts/utils/PathRange.ts

```diff
@@ -42,23 +42,27 @@ const generatePathRange = (root: Node, startNode: Node, startOffset: number, end
 const resolvePath = (root: Node, path: number[]): Optional<{node: Node; offset: number}> => {
   const nodePath = path.slice();
   const offset = nodePath.pop();
-  return Arr.foldl(nodePath, (optNode: Optional<Node>, index: number) => optNode.bind((node) => Optional.from(node.childNodes[index])), Optional.some(root)).bind((node) => {
-    if (Utils.isText(node) && offset >= 0 && offset <= node.data.length) {
-      return Optional.some({ node, offset });
+  const resolvedNode = Arr.foldl(nodePath, (optNode: Optional<Node>, index: number) => optNode.bind((node) => Optional.from(node.childNodes[index])), Optional.some(root));
```

Comment:
This was how the existing code was structured but would it be more readable to have each parameter on a separate line?